plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableDataStoreTests.php

```diff
@@ -906,15 +898,15 @@ public function test_cot_query_pagination(): void {
 		$query = new OrdersTableQuery();
 		$this->assertCount( 30, $query->orders, 'If no limits are specified, we fetch all available orders.' );
 
-		$query = new OrdersTableQuery( array( 'limit' => -1 ) );
+		$query = new OrdersTableQuery( array( 'limit' => - 1 ) );
 		$this->assertCount( 30, $query->orders, 'A limit of -1 is equivalent to requesting all available orders.' );
 
-		$query = new OrdersTableQuery( array( 'limit' => -10 ) );
+		$query = new OrdersTableQuery( array( 'limit' => - 10 ) );
```

Comment:
It's a matter of personal pref since we don't have any rules around this, but `- 10` seems less readable to me than `-10` (the latter being the convention I am most used to ... but that could just be me 🙃).

Reply:
Either way is fine to me, so happy to apply your change.

Reply:
Agreed with Barry here, I'd say that a leading space should be applied to binary minus (`foo - bar`, it's an operator) but not to unary minus (`-foo`, it's part of the value).